builds/misc/templates/base-image-publish/base-image-setup-librocksdb.yaml

```diff
@@ -0,0 +1,18 @@
+steps:
+- pwsh: |
+    $artifactLocale = "$(Build.SourcesDirectory)/librocksdb.so"
+    Invoke-WebRequest -Uri $("https://edgebuild.blob.core.windows.net/librocksdb/librocksdb.so.$(arch)") -OutFile $artifactLocale
```

Comment:
We will need to remember to update this file whenever we increase the rocksdb version

Reply:
Yup. 
TODO: Create a documentation about this tool (and address this concern).